src/NuGet.Services.EndToEnd/Support/Clients/Clients.cs

```diff
@@ -53,5 +65,15 @@ public static Clients Initialize()
                 registration,
                 nuGetExe);
         }
+
+        private static IAzureManagementAPIWrapper GetAzureManagementAPIWrapper(TestSettings testSettings)
```

Comment:
IAzureManagementAPIWrapper  is defined in a different package. Not trivial to change